### PR TITLE
[feature1]new pull request

### PR DESCRIPTION
**Pull request by opensource206**
1.created new file
2.created new folder

### CODE REVIEW

File: newfile [hunk 1]
```diff
@@ -0,0 +1 @@
+local commit in newfile
```
Comment (opensource206) [inline newfile:1]: this message could be better

**Comment by hacker748206** [comment]
but of course this message looks ok to me

**Comment by hacker748206** [comment]
i am going to approve this

**Review by hacker748206 — APPROVED** [review]

**Comment by opensource206** [comment]
just a pull request merge

**Pull request closed without merging**